test/extended/builds/multistage.go

```diff
@@ -37,12 +39,10 @@ COPY --from=test /usr/bin/curl /test/
 			}
 		})
 
-		g.It("should succeed [Conformance]", func() {
+		g.It("should succeed OCP-22466 OCP-22468 OCP-22469 [Conformance]", func() {
```

Comment:
please remove this change - this text appears in the test case description which developers use to run individual tests

Reply:
Removed